CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-11-05
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.28 is GA and 1.25 will not be deprecated until the end of January 2024.  This means that 4 versions will be GA during this time.
```

Comment:
Feature

No need for mention of 4 versions or 1.25 deprecation, they are not related. 1.25 will be deprecated in Jan when it has 12+ months